caddyhttp/browse/browse_test.go

```diff
@@ -300,7 +300,7 @@ func TestBrowseJson(t *testing.T) {
 		req.Header.Set("Accept", "application/json")
 		rec := httptest.NewRecorder()
 
-		code, err := b.ServeHTTP(rec, req)
+		code, _ := b.ServeHTTP(rec, req)
```

Comment:
Well, in my case the CI test where complaining about my pull request, pointing me to what @elcore mentioned. I would not have been picky about the style of your code, if your CI tools hadn't forced me ;)